src/nplinker/loader.py

```diff
@@ -54,12 +60,22 @@ class DatasetLoader:
     OR_CANOPUS = "canopus_dir"
     OR_MOLNETENHANCER = "molnetenhancer_dir"
 
-    def __init__(self, config: Dynaconf):
+    def __init__(self, config: Dynaconf) -> None:
         """Initialize the DatasetLoader.
 
         Args:
-            config: A Dynaconf object that contains the configuration settings. Check the
-                `nplinker.config` module for more information.
+            config: A Dynaconf object that contains the configuration settings.
+
+        Examples:
+            >>> from nplinker.config import load_config
+            >>> from nplinker.loader import DatasetLoader
+            >>> config = load_config("nplinker.toml")
+            >>> loader = DatasetLoader(config)
+            >>> loader.load()
+
+        See Also:
```

Comment:
```suggestion
        See also:
```

Reply:
Ruff requires that all section headers in a docstring should be [capitalized](https://docs.astral.sh/ruff/rules/capitalize-section-name/). So let's use `See Also:`.